backend/fms_core/serializers.py

```diff
@@ -268,3 +271,13 @@ class Meta:
         model = Group
         fields = ("id", "name", "permissions")
         depth = 1
+
+class ProjectSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Project
+        fields = "__all__"
+
+class ProjectExportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Project
+        fields = "__all__"
```

Comment:
This is the data that would be in the export file
![image](https://user-images.githubusercontent.com/1617741/128362593-4e69cb54-08fb-49b3-ae47-39981b0b17e2.png)

There are probably some irrelevant fields and/or some fields with foreign id's that could display information formatted in a more human-friendly way :P